book/website/collaboration/ppie-events/ppie-events-comms.md

```diff
@@ -0,0 +1,363 @@
+(cl-ppie-events-comms)=
+# Planning Communications for PPIE Events
+Having a good plan on how and when to communicate around your event is always important, but it can be even more relevant when inviting people with lived experience to your event as part of supporting attendees who are new to research events.
+
+This section provides guidance on what to consider when developing a communications plan for events involving people with lived experience. 
+It will not be able to replace an in-depth communications plan, but instead draws attention to what additional considerations may be needed to help make your event inclusive of people with lived experience. 
+Please refer to the {ref}`Planning your Event<cl-ppie-events-planning>` section for more information about planning your event. 
+You can find a checklist at the end of this page for what key pieces of information need to be included in your event's core communications.
+
+For this chapter, "Communications" refers to the information you share with potential attendees, the timings and schedule on which you communicate it, the channels you will use, and the content and information about the event. 
+
+When developing a communications strategy, in addition to identifying the goals for your event, you will need to consider:
+- Who are you trying to reach?
```

Comment:
```suggestion
- Who are you trying to reach more broadly?
```
Just a suggestion - is this a broader question about reach, asking about attendees with lived experiences as a subset?

Reply:
It's more about the audience in general - the suggestions are more guidance for how a communications strategy can be created - i.e. who is the target audience for a specific piece of information.